### PR TITLE
added techreborn:ores to c:ores

### PR DESCRIPTION
**Pull request by Ayutac**

### CODE REVIEW

**Comment by drcrazy** [comment]
Should it be blocks rather then items? I suspect Diggus Maximus check block tags to do a vein mining.

**Comment by Ayutac** [comment]
that would require a bit more work. I can look into it after my TR compiles again

**Comment by drcrazy** [comment]
@Ayutac Wanna complete this?

File: src/datagen/groovy/techreborn/datagen/tags/TRBlockTagProvider.groovy [hunk 1]
```diff
@@ -48,5 +52,9 @@ class TRBlockTagProvider extends FabricTagProvider.BlockTagProvider {
 			.addOptionalTag(BlockTags.AXE_MINEABLE.id())
 			.addOptionalTag(FabricMineableTags.SHEARS_MINEABLE.id())
 			.addOptionalTag(FabricMineableTags.SWORD_MINEABLE.id())
+
+		TRContent.Ores.values().each { ore ->
+			getOrCreateTagBuilder(TagKey.of(Registry.BLOCK_KEY, new Identifier("c", "ores"))).add(ore.asBlock())
+		}
```
Comment (velna) [inline src/datagen/groovy/techreborn/datagen/tags/TRBlockTagProvider.groovy:58]:
```	
	TRContent.Ores.values().each {
			getOrCreateTagBuilder(ConventionalBlockTags.ORES)
				.add(it.block)
	}
```
Will do the trick.

**Comment by velna** [comment]
> Should it be blocks rather then items? I suspect Diggus Maximus check block tags to do a vein mining.

This patch will also fix the problem that the Jackhammer breaks TR ore blocks.

**Comment by drcrazy** [comment]
PR Obsolete. BlockTags are Implemented in TR datagen.

**Pull request closed without merging**